.travis.yml

```diff
@@ -54,9 +57,28 @@ script:
   - if [ "$DOCKER" == true ]; then bash ./scripts/travis/build-docker-images.sh ; else echo 'skipping docker images'; fi
   - if [ "$ES_INTEGRATION_TEST" == true ]; then bash ./scripts/travis/es-integration-test.sh ; else echo 'skipping elastic search integration test'; fi
   - if [ "$HOTROD" == true ]; then bash ./scripts/travis/hotrod-integration-test.sh ; else echo 'skipping hotrod example'; fi
+  - if [ "$WINDOWS" == true ]; then bash ./scripts/travis/build-windows.sh ; else echo 'skipping windows'; fi
 
 after_success:
   - if [ "$COVERAGE" == true ]; then travis_retry goveralls -coverprofile=cover.out -service=travis-ci || true ; else echo 'skipping coverage'; fi
 
 after_failure:
   - if [ "$CROSSDOCK" == true ]; then make crossdock-logs ; else echo 'skipping crossdock'; fi
+
+deploy:
```

Comment:
I am not familiar with travis deploy workflow - where do these files end up?  And do we need a different naming conventions? E.g. https://prometheus.io/download/:
```
prometheus-2.2.1.darwin-amd64.tar.gz
prometheus-2.2.1.linux-amd64.tar.gz
prometheus-2.2.1.windows-amd64.tar.gz
```

Also, since these are archive files, do we want to bundle all binaries into one archive?

Reply:
also, we should change these commands to only build and deploy these binaries on landing the PR rather than every commit (unless this already does this)

Reply:
> We could also add `branch: master` to further pin down when it deploys.

[I don't think that's what we want](https://docs.travis-ci.com/user/deployment/releases/):

> GitHub Releases uses git tags. If the build commit does not have any tags, one will be created in the form of `untagged-*`, where `*` is a random hex string.

Reply:
@jpkrohling I think that's only if you have `on.branch: true`. I was thinking that if you had multiple `on` options it would only deploy if all conditions were met. I'll look into it.

Reply:
I verified `When all conditions specified in the 'on:' section are met, your build will deploy.` So if we have `on.tags: true` and `on.branch: master` it will only deploy when it's a commit on master that has a tag already.